src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileStatusFields.scala

```diff
@@ -15,11 +15,14 @@ object FileStatusFields {
   case class FileStatus(fileId: UUID, statusType: String, statusValue: String)
 
   case class AddFileStatusInput(fileId: UUID, statusType: String, statusValue: String)
+  case class AddMultipleFileStatusInput(statuses: List[AddFileStatusInput])
 
   implicit val FileStatusType: ObjectType[Unit, FileStatus] = deriveObjectType[Unit, FileStatus]()
   implicit val AddFileStatusInputType: InputObjectType[AddFileStatusInput] = deriveInputObjectType[AddFileStatusInput]()
+  implicit val AddMultipleFileStatusInputType: InputObjectType[AddMultipleFileStatusInput] = deriveInputObjectType[AddMultipleFileStatusInput]()
 
   implicit val FileStatusInputArg: Argument[AddFileStatusInput] = Argument("addFileStatusInput", AddFileStatusInputType)
+  implicit val MultipleFileStatusInputArg: Argument[AddMultipleFileStatusInput] = Argument("addMultipleFileStatusInput", AddMultipleFileStatusInputType)
```

Comment:
Should this be re-named: `MultipleFileStatusesInputArg`

Reply:
I've renamed the variable and the argument name